BankSDK/GiniBankSDK/Sources/GiniBankSDK/Resources/GiniColorScheme.swift

```diff
@@ -0,0 +1,256 @@
+//
+//  GiniColorSchemeV2.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+import UIKit
+import GiniCaptureSDK
+
+public struct GiniColorScheme {
+    struct Bg {
+        var primary: GiniColor
+        var secondary: GiniColor
+    }
+
+    struct ActionSheets {
+        var buttonsBg: GiniColor
+        var cancelButtonBg: GiniColor
+    }
+
+    struct Alert {
+        var bg: GiniColor
+        var divider: GiniColor
+    }
+
+    struct NavigationBar {
+        var bg: GiniColor
+        var action: GiniColor
+        var navigation: GiniColor
+    }
+
+    struct Content {
+        var navigation: GiniColor
+        var text: GiniColor
+    }
+
+    struct BottomBar {
+        var bg: GiniColor
+        var border: GiniColor
+    }
+    
+    struct Card {
```

Comment:
[clarification] What is card for? We should use better names for our internal structs.

Reply:
renamed to Container